### PR TITLE
refactor(protocol-designer): move deck setup folder to correct pg

### PR DESCRIPTION
**Pull request by jerader**
closes AUTH-674

# Overview

When i made the deck setup foundational components, i added it to the `ProtocolOverview` page but `StartinDeckState` actually has its own page. so this pr moves the components over.

## Test Plan and Hands on Testing

turn the redesign ff on and go through the file creation wizard. once you get on the protocol overview page and see the deck setup button, click on that and it should go to the correct `/startedDeckState` route to view the deck map. We probably need to refactor that component a bit down the road so we can utilize the same deck map for when displaying and adding the protocol steps. But i think it is fine like this for now

## Changelog

move deck setup folder to the starting deck state folder and fix test

## Review requests

see test plan

## Risk assessment

low

### CODE REVIEW

File: protocol-designer/src/pages/StartingDeckState/index.tsx [hunk 1]
```diff
@@ -1,8 +1,266 @@
...
+      >
+        cancel
+      </PrimaryButton>
+      <PrimaryButton
+        onClick={() => {
+          navigate('/overview')
+        }}
+      >
+        save
+      </PrimaryButton>
```
Comment (koji) [inline protocol-designer/src/pages/StartingDeckState/index.tsx]: no diff between cancel button and save button?

File: protocol-designer/src/pages/StartingDeckState/index.tsx [hunk 2]
```diff
@@ -1,8 +1,266 @@
...
+      {zoomIn != null ? (
+        //  TODO(ja, 8/6/24): still need to develop the zoomed in slot
+        <LegacyStyledText>you zoomed in on the slot!</LegacyStyledText>
+      ) : (
+        <Flex
+          maxWidth={robotType === FLEX_ROBOT_TYPE ? '130vh' : '100vh'}
```
Comment (koji) [inline protocol-designer/src/pages/StartingDeckState/index.tsx]:
```suggestion
          maxWidth={robotType === FLEX_ROBOT_TYPE ? '130vw' : '100vw'}
```

File: protocol-designer/src/pages/StartingDeckState/index.tsx (hunk 1 above)
Comment (jerader) [reply protocol-designer/src/pages/StartingDeckState/index.tsx]: true, these were placeholders but looking at designs, they should be one button lol. i'll fix it

File: protocol-designer/src/pages/StartingDeckState/index.tsx (hunk 2 above)
Comment (jerader) [reply protocol-designer/src/pages/StartingDeckState/index.tsx]: wow i actually never knew what vh stood for and didn't realize it is viewHeight and that there is a viewWidth 😆

**Review by koji — APPROVED** [review]
lgtm

**Pull request merged**